packages/components/src/combobox-control/index.js

```diff
@@ -280,8 +280,14 @@ function ComboboxControl( {
 							onHover={ setSelectedSuggestion }
 							onSelect={ onSuggestionSelected }
 							scrollIntoView
-							renderSuggestion={ ( { suggestion } ) =>
-								renderOption?.( { option: suggestion } )
+							renderSuggestion={
+								typeof renderOption === 'function'
+									? ( { suggestion } ) => {
+											renderOption( {
```

Comment:
This is almost ready. Now, we need to add a `return` here. Otherwise, the anonymous function doesn't return anything, and the `With Render Option` case does not work.

Reply:
🤦‍♂️ Thanks